cmd/terramate/cli/github/client.go

```diff
@@ -91,6 +93,44 @@ func (c *Client) Commit(ctx context.Context, repository, sha string) (*Commit, e
 	return &commit, nil
 }
 
+// PullReviews returns the list of reviews of the given pull request number.
+func (c *Client) PullReviews(ctx context.Context, repository string, nr int) (Reviews, error) {
+	if err := validateRepo(repository); err != nil {
+		return nil, err
+	}
+	const perPage = 100
+	urlStr := fmt.Sprintf("%s/repos/%s/pulls/%d/reviews", c.baseURL(), repository, nr)
```

Comment:
I thought you wanted to get the list of reviewers from the ones that was requested (which also includes the ones that made reviews which returns this endpoint). I mean `requested_reviewers` field in https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request

Reply:
The `requested_reviewers` becomes empty after a review is submitted so by the time that PR is ready to be merged (or changes applied to the IaC) then it contains an empty list.

Reply:
Have a look here: https://api.github.com/repos/terramate-io/terramate/pulls/1175
This PR was reviewed.

Reply:
Awesome! That's great to know :people_hugging: